examples/minimal-configuration.nix

```diff
@@ -0,0 +1,22 @@
+{ config, pkgs, lib, ... }: {
+  imports = [
+    <nix-bitcoin/modules/nix-bitcoin.nix>
+    <nix-bitcoin/modules/generate-secrets.nix>
```

Comment:
Shouldn't this be `<nix-bitcoin/modules/secrets/generate-secrets.nix>`? I'm getting
```
error: file 'nix-bitcoin/modules/generate-secrets.nix' was not found in the Nix search path (add it using $NIX_PATH or -I), at /home/nix-bitcoin/nix-bitcoin/examples/minimal-configuration.nix:4:5
```

Reply:
Thanks, this was an untested last-minute change. Fixed.